tests/acceptance/features/coreApiWebdavMove1/moveFolderToBlacklistedName.feature

```diff
@@ -51,9 +50,6 @@ Feature: users cannot move (rename) a folder to a blacklisted name
     And user "Brian" has been created with default attributes and without skeleton files
     And user "Brian" has created folder "/testshare"
     And user "Brian" has created folder "/FOLDER"
-    # Note: we have to write JSON for the value, and to get a backslash in the double-quotes we have to escape it
-    # The actual regular expressions end up being .*\.ext$ and ^bannedfilename\..+
-    And the administrator has updated system config key "blacklisted_files_regex" with value '[".*\\.ext$","^bannedfilename\\..+","containsbannedstring"]' and type "json"
```

Comment:
The whole test scenario is useless without the ability do somehow define the blacklisted files regex
We could put a "ToDo' comment in the scenario to say that a step is needed that defines the blacklisted files regex, and that step cannot be written yet.